src/parse.y

```diff
@@ -206,8 +206,14 @@ option		:  TOK_OUTFILE '=' NAME
 			{ yyclass = xstrdup(nmstr); }
 		|  TOK_HEADER_FILE '=' NAME
 			{ headerfilename = xstrdup(nmstr); }
-	    |  TOK_TABLES_FILE '=' NAME
-            { tablesext = true; tablesfilename = xstrdup(nmstr); }
```

Comment:
I agree, an extra strdup() would be better than all the extra is_xxx_alloc guards.